lib/index.js

```diff
@@ -236,14 +236,15 @@ const KiteAPI = {
     );
   },
 
-  requestRelatedCode(editor, filename, line, selection) {
```

Comment:
minor nit: consider grouping editor and editor_path?

Reply:
I prefer to keep it separate here, since it would be a non-breaking change, both from the editor and Kited side.

Reply:
Well, when we add selection, the function signature would be `requestRelatedCode(editor, filename, line, editor_install_path, selection)` or something. It's not a huge deal, but it seems slightly better to group `(editor, editor_install_path, filename, line)` while we don't officially support code nav. I don't think anything would have to change about the json sent to kited

Reply:
Given that we hard lock Kite API to each editor plugin, I'm okay with making this change. 

Generally, I prefer to add new params to the end of a released API; that way I can release the change and use the new version in one consumer of the API without needing to change how it's called in other consumers until desired.